Project.toml

```
@@ -37,11 +35,10 @@ IntervalSets = "0.7"
 InverseFunctions = "0.1.5"
 MacroTools = "0.5"
 Markdown = "1"
-Requires = "0.5, 1.0"
 StaticArrays = "1"
 StructArrays = "0.6"
 Unitful = "1"
-julia = "1.6"
+julia = "1.10"
```

IMO this should be
```suggestion
julia = "1.9"
```
so also people on Julia 1.9 can benefit from this fix in case the Test dependency caused circular loading for them.